blaze-client/src/test/scala/org/http4s/client/blaze/ClientTimeoutSpec.scala

```diff
@@ -178,7 +178,7 @@ class ClientTimeoutSpec extends Http4sSpec {
       // if the requestTimeout is hit then it's a TimeoutException
       // if establishing connection fails first then it's an IOException
 
-      // The expected behaviour is that the requestTimeout will happen first, but fetchAs will additionally wait for the IO.sleep(2.seconds) to complete.
+      // The expected behaviour is that the requestTimeout will happen first, but fetchAs will additionally wait for the IO.sleep(1000.millis) to complete.
```

Comment:
So the `requestTimeout` isn't actually respected, i.e. given that `c.fetchAs[String](FooRequest).unsafeRunTimed(1500.millis)` fails after `1000 ms`, not `50 ms`, @RafalSumislawski?

Reply:
> An acquire operation of cats' Bracket/Resource is uncancelable

is the answer to my question, per https://github.com/http4s/http4s/issues/2386#issuecomment-475782999?